src/windows/VibrationProxy.js

```diff
@@ -35,26 +35,74 @@ function tryDoAction(actionName, success, fail, args, action) {
         }
 
         action(args);
-        success();        
+        success && success();        
```

Comment:
Proxy calls are always called with a valid success + fail callback function.  No need for the extra checks.